### PR TITLE
Make sure we freeze ruby-maven, maven-tools and jar-dependencies version

### PR DESCRIPTION
**Pull request by ph**
Fix the maven gems to their latest known working version.

### CODE REVIEW

File: logstash.gemspec [hunk 1]
```diff
@@ -37,13 +37,17 @@ Gem::Specification.new do |gem|
 
   # Plugin manager dependencies
 
-  # jar-dependencies 0.1.2 is included in jruby 1.7.6 no need to include here and
-  # this avoids the gemspec jar path parsing issue of jar-dependencies 0.1.2
+  # Currently there is a blocking issue with the latest (3.1.1.0.9) version of
```
Comment (purbon) [inline logstash.gemspec]: can you point that to the actual logstash issue?

Comment (ph) [inline logstash.gemspec]: look below?

Comment (ph) [inline logstash.gemspec]: line 46

Comment (purbon) [inline logstash.gemspec]: jajaja, ok! sorry.

**Comment by colinsurprenant** [comment]
can you write an explicit comment? just referring to an ticket is not very helpful.

**Comment by purbon** [comment]
Agreed, explicit comment can be more helpful.

**Comment by colinsurprenant** [comment]
disregard my previous comment... explicit comment is already there... I was mesmerized by the diff stuff

**Comment by colinsurprenant** [comment]
LGTM

**Comment by ph** [comment]
@purbon @colinsurprenant do you need some :coffee:  ? :)

**Comment by colinsurprenant** [comment]
you'll have problem cherrypicking the .lock changes into the bundle branch

**Comment by purbon** [comment]
Testing is good, LGTM

**Pull request closed without merging**